### DISCUSSION TITLE
OpenAPI YAML/JSON

### DISCUSSION THREAD

**Discussion by SaschaKober** [marlon360/rki-covid-api#248] [General]
Hey, gibt es eine openapi description Datei für die API? :)

**Comment by marlon360** [comment]
Hey!
Die gibt es aktuell noch nicht. Aber du kannst gerne ein Feature Request in den Issues aufmachen und auch gerne einen PR machen, wenn du möchtest :)

**Comment by SaschaKober** [comment]
[CoronaZahlen-1.0.0-resolved_updated.zip](https://github.com/marlon360/rki-covid-api/files/6800443/CoronaZahlen-1.0.0-resolved_updated.zip)

Kannst du dir mal anschauen :)
